outside.go

```diff
@@ -352,9 +352,7 @@ func (f *Interface) handleRecvError(addr *udp.Addr, h *header.H) {
 	// We delete this host from the main hostmap
 	f.hostMap.DeleteHostInfo(hostinfo)
```

Comment:
I think it would be more correct to call `f.closeTunnel(hostinfo, true)` since this is missing a few parts of removing a tunnel currently.

Reply:
Good call, PR updated.